packages/front-end/components/Auth/CreateOrganization.tsx

```diff
@@ -162,7 +180,11 @@ export default function CreateOrganization(): ReactElement {
                         onClick={() => {
                           joinOrgFormSubmit(org);
                         }}
-                        disabled={org.currentUserIsPending || false}
+                        disabled={
+                          org.currentUserIsPending ||
+                          org.currentUserIsMember ||
```

Comment:
Can remove this line as it wouldn't be in joinableOrg if they are already a member.